examples/file-feed/complete/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Anonymous+Pro|Source+Code+Pro" rel="stylesheet">
```

Comment:
Probably we should just not use third-party fonts and set `font-family: sans-serif`, it's usually good enough.